### PR TITLE
Odds

### PR DESCRIPTION
**Pull request by GAylabea**
Program file is functioning - prints integers.

### CODE REVIEW

File: Mathmagician.Tests/IntegerTests.cs [hunk 1]
```diff
@@ -34,13 +34,13 @@ public void IntegerEnsureUserInputIsAnInteger()
...
-            int answerModTwo = ModuloTwo.ModuloTwo(10);
-            Assert.AreEqual((0), answerModTwo);
-        }
+        //[TestMethod]
+        //public void IntegerModuloTwo()
+        //{
+        //    Integer ModuloTwo = new Integer();
+        //    int answerModTwo = ModuloTwo.ModuloTwo(10);
+        //    Assert.AreEqual((0), answerModTwo);
+        //}
```
Comment (BlaiseGratton) [inline Mathmagician.Tests/IntegerTests.cs]: Try to remove superfluous commented out blocks

**Comment by SandraJones** [comment]
OddTest.cs small typo on line 23 may need to be corrected.

File: Mathmagician/Odds.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        }
+
+        //virtual property list says it can be overridden by a child class such as odds  
+        public override List<int> IntegerList { get; set; }
+        public List<int> OddList { get; private set; }
+
+        public override List<int> GenerateList(int listLength)
+        {
+
+            for (var i = 0; OddList.Count != listLength; i++)
```
Comment (BlaiseGratton) [inline Mathmagician/Odds.cs:22]: `OddList.Count != listLength` is a clever condition, I like it

**Comment by BlaiseGratton** [comment]
Once you remove commented out code blocks, 👍

**Comment by SandraJones** [comment]
:+1:

**Pull request closed without merging**